crawl-ref/source/mon-place.cc

```diff
@@ -74,11 +74,6 @@ static vector<bool> vault_mon_bands;
 #define VAULT_MON_BANDS_KEY   "vault_mon_bands"
 #endif
 
-// proximity is the same as for mons_place:
-// 0 is no restrictions
-// 1 attempts to place near player
-// 2 attempts to avoid player LOS
-
```

Comment:
I'm in favour of removing this pseudo-enum. But I'm curious where the code is that this used to refer to, and how it was updated to not need this pseudo-enum anymore.

Reply:
These comments actually go back to the initial import in 673bdae75. I'm not sure specifically how these values were adapted into the proximity_type that mgen_data now uses, but it appears that happened back in Linley crawl.